specs/deposits.md

```diff
@@ -349,13 +349,16 @@ the genesis file will be located in the `deployedBytecode` field of the build ar
 
 #### Ecotone L1Block upgrade
 
-The L1 Attributes Predeployed contract, `L1Block.sol`, is upgraded in the Ecotone upgrade.
+The L1 Attributes Predeployed contract, `L1Block.sol`, is upgraded as part of the Ecotone hardfork.
+The version is incremented to `1.2.0` and several new storage slots are used for:
 
-- `overhead`: this getter function now reverts (TBD). Storage remains the same.
-- `scalar`: this getter function now reverts (TBD). Storage remains the same.
```

Comment:
What happens when a user calls one of these two getters? How does the behavior change?

Reply:
yeah it still reverts, we should leave this in?

Reply:
I removed this from here as I believe those getters are [part of the GasPriceOracle contract](https://github.com/ethereum-optimism/optimism/pull/8726/files#diff-7eeb2cdd3a766512a162b2921f45ae64a3ae91f99d7d408ca11defb6142f4ba5R66-R84) and not the L1Block contract.

Calling either `GasPriceOracle::overhead` or `GasPriceOracle::scalar` will both revert post Ecotone. Do you think we should specify this here or in [predeploys](https://github.com/ethereum-optimism/optimism/blob/develop/specs/predeploys.md#gaspriceoracle)?

Reply:
excellent point, yes should be part of predeploys

Reply:
Sounds good 👍 Just checked and it looks like it has already been specified [there](https://github.com/ethereum-optimism/optimism/pull/8602/files#diff-efcf302ad36b96db88eff5ac88ea97e5ec976823f109b71da706b153e102b53aR234-R253). 

Let me know if ya'll think it's worth stating here too.

Reply:
Added a TODO comment on the specs PR, to clarify it more in the specs you pointed out. This PR is good to go.